src/jdk.jshell/share/classes/jdk/jshell/SnippetMaps.java

```diff
@@ -186,6 +194,13 @@ String fullClassNameAndPackageToClass(String full, String pkg) {
         if (Stream.concat(Stream.of("java.lang"), pkgs).anyMatch(pkg::equals)) {
             return full.substring(pkg.length() + 1);
         }
+        Stream<String> mods = importSnippets()
```

Comment:
Should this variable be called `mods` ? AFAIU, it lists all the packages exported by a module.

Reply:
Thanks, attempted to improve here:
https://github.com/openjdk/jdk/pull/23801/commits/4f9575564c3761b251288c3dec0132cbe0a16139